WordPress/Classes/ViewRelated/People/PersonViewController.swift

```diff
@@ -179,13 +124,33 @@ final class PersonViewController: UITableViewController {
         // not being timely initialized.
         //
         //
-        if isFullnamePrivate == true && fullnameSection == indexPath.section && fullnameRows.contains(indexPath.row) {
+        if isFullnamePrivate == true && (indexPath == firstNameIndexPath || indexPath == lastNameIndexPath) {
```

Comment:
I found this line a bit difficult to parse. Would it make sense to encapsulate what it does in a private method, so that here, at the call site, the code would read something like this?:
`
if cellIsHidden(indexPath) { return ... } 
`

or 
` guard cellIsNotHidden(indexPath) else { return CGFloat.leastNormalMagnitude }
`